bundles/core/src/main/java/com/adobe/cq/wcm/core/components/commons/link/Link.java

```diff
@@ -64,7 +62,7 @@ default boolean isValid() {
     }
 
     /**
-     * The link URL, supports context path and vanity paths.
+     * The link URL, supports context path and escaping.
```

Comment:
I'd go with: Returns the link URL ...

Reply:
The @return tag should make this clear. It's also not aligned with other interfaces like OEmbedClient for example.